changelog/september2023/2023-09-07-kubernetes-changed-kapsule-and-kosmos-clusters.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: Kapsule and Kosmos clusters have a new limit on first offer: 150 nodes maximum
+status: changed
+author:
+    fullname: 'Join the #k8s channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2023-09-07
+category: containers
+product: kubernetes
+---
+
+Effective September 1st 2023. To go beyond 150 nodes, Scaleway offers a range of dedicated control planes to cater to a wide range of use-cases. Refer to our [Kubernetes control plane offers overview](https://www.scaleway.com/en/docs/containers/kubernetes/reference-content/kubernetes-control-plane-offers/) for more information.
```

Comment:
```suggestion
Starting September 1st 2023, Kapsule and Kosmos clusters are limited to 150 nodes maximum on first offer. Scaleway offers a range of dedicated control planes to cater to a wide range of use-cases, to go beyond 150 nodes. Refer to our [Kubernetes control plane offers overview](/containers/kubernetes/reference-content/kubernetes-control-plane-offers/) for more information.
```

Reply:
I don't know if this is what was meant. @tgenaitay can you clear it up for us? 🙏🏻

Reply:
@ldecarvalho-doc
As a non native speaker, it sounds like the same to me. The point is: to go beyond the new limit, DCP are here.

Reply:
LGTM!